rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23_ckxaut.md

```diff
@@ -0,0 +1,41 @@
+# Proposal to upgrade the ledger suite orchestrator canister to add ckXAUT
```

Comment:
this file was simply rename in this [commit](https://github.com/dfinity/ic/pull/1075/commits/975a00a588a8e2aad3adfcd578c1fe916ca11106).